client/packages/system/src/Report/api/hooks/useReportList.ts

```diff
@@ -39,15 +39,12 @@ export const useReportList = ({
     nodes: ReportRowFragment[];
     totalCount: number;
   }> => {
-    const filter =
-      context || subContext
-        ? {
-            context: context ? { equalTo: context } : null,
-            subContext: subContext ? { equalTo: subContext } : null,
-          }
-        : null;
     const query = await reportApi.reports({
-      filter: { ...filterBy, ...filter },
+      filter: {
+        ...filterBy,
+        ...(context ? { context: { equalTo: context } } : null),
+        ...(subContext ? { subContext: { equalTo: subContext } } : null),
```

Comment:
this is a much nicer way to construct the params!
I'm not sure the scenario of having a subContext without a context is valid - but there's no reason to change that existing logic, nor to have a test for it in here